.github/workflows/continous-integration.yml

```diff
@@ -162,15 +162,33 @@ jobs:
         user: __token__
         password: ${{ secrets.PYPI_TOKEN }}
 
-    - name: Publish Release Notes 🗞
-      if: env.IS_TAG_BUILD
+    - name: Notify Slack & Publish Release Notes 🗞
```

Comment:
We should leave the `Publish Release Notes` step as it is and add new steps for sending Slack notifications that depend on the `Publish Release Notes` step.